### ISSUE TITLE
File field 500 error

### ISSUE THREAD

**Issue by lylebrown** [benjaminjonard/koillection#430]
I'm having an issue where attempting to upload a file via a file field results in a 500 error.

I am on the latest version as of posting this issue, running with docker-compose. 

My var/logs/ directory is empty, and the log reported up to docker doesn't contain any information beyond the initial start messages, so I'm not sure what other information I can provide on this one.

**Comment by benjaminjonard** [comment]
Could you try to activate debug mode ?  (add APP_DEBUG=1 in your docker-compose file like below)

```
    koillection:
        image: koillection/koillection
        container_name: koillection
        restart: unless-stopped
        ports:
            - 80:80
        environment:
            - APP_DEBUG=1
            - DB_DRIVER=pdo_pgsql (or pdo_mysql)
            - DB_NAME=koillection
            - DB_HOST=db
            - DB_PORT=5432 (3306 for mysql)
            - DB_USER=root
            - DB_PASSWORD=root
            - DB_VERSION=12.2
            - PHP_TZ=Europe/Paris
            - HTTPS_ENABLED=1
``` 

If after that there is still nothing in var/logs, could you share a screenshot of the error please ?

**Comment by lylebrown** [comment]
So it looks like it's unable to create the directory within uploads. Still nothing in var/logs, but the error appeared in the browser.

Struggling to understand why that would be. I ran a shell in the container as root, and I was able to create files and directories in that uploads directory. Also tried a shell in the container as koillection, and that worked as well. I put 777 permissions on the directory on the filesystem (outside of the container) for testing, but it's still not working.

Here's the detailed error:
![image](https://user-images.githubusercontent.com/6559258/178775394-6743ffa7-058a-44f9-b276-4aca4f0cc04f.png)
